packages/logging-winston/src/index.js

```diff
@@ -148,7 +148,7 @@ LoggingWinston.prototype.log = function(levelName, msg, metadata, callback) {
     metadata = {};
   }
 
-  if (!this.levels_[levelName]) {
+  if (typeof this.levels_[levelName] === 'undefined') {
```

Comment:
nit: I think a comparison of the form `this.levels_[levelName] === undefined` would work just as well here. I have pushed a change to this effect. Let me know if you disagree.